net-wireless/eiwd/eiwd-1.6.1.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit flag-o-matic
+
+MY_PV="$(ver_rs 2 '-')"
+MY_P="${PN}-${MY_PV}"
+
+DESCRIPTION="iwd without dbus"
+HOMEPAGE="https://github.com/dylanaraps/eiwd"
+SRC_URI="https://github.com/dylanaraps/eiwd/releases/download/${MY_PV}/${MY_P}.tar.xz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+client +monitor ofono +resolvconf +system-ell wired"
+
+DEPEND="system-ell? ( >=dev-libs/ell-0.31 )"
+RDEPEND="${DEPEND}
+	resolvconf? ( || ( net-dns/openresolv net-misc/dhcpcd ) )
```

Comment:
Don't use USE flags for runtime deps and small files

Reply:
Done